src/processors/docx/docx_extractor.py

```
@@ -0,0 +1,294 @@
+            full_content = "\n\n".join(content_parts)
+            
+            # Clean and normalize
+            cleaned_content = self._clean_docx_text(full_content)
+            
+            if not cleaned_content.strip():
+                logger.warning(f"No content extracted from {file_path}")
+                return ""
+            
+            return cleaned_content
+            
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)